packages/extension/src/index.ts

```diff
@@ -44,8 +44,15 @@ const plugin: JupyterFrontEndPlugin<void> = {
         });
     }
 
-    const { commands } = app;
-    const api = new CommandBridge('jupyterlab', { commands });
+    const api = {
+      execute(command: string, args: ReadonlyPartialJSONObject) {
+        commands.execute(command, args);
+      },
+      listCommands() {
+        console.log('Commands: ', commands.listCommands());
```

Comment:
Looks like we could return the result of `commands.listCommands()` here directly, as it may be useful to the host page?